src/zenml/config/pipeline_configurations.py

```diff
@@ -42,6 +43,25 @@ class PipelineConfiguration(PipelineConfigurationUpdate):
 
     name: str
     enable_cache: bool
+    enable_artifact_metadata: bool
+
+    @root_validator(pre=True)
+    def _fill_missing_values(cls, values: Dict[str, Any]) -> Dict[str, Any]:
```

Comment:
Can we just give it a default value of `True` or `False` and remove this method entirely?